typescript/schemas/README.md

```diff
@@ -0,0 +1,3 @@
+# @foxglove/schemas
+
+This package contains TypeScript type definitions for the Foxglove schemas.
```

Comment:
this description is missing a mention of json schemas?

Reply:
updated + examples